bodhi/client/bindings.py

```diff
@@ -71,6 +71,9 @@ def __init__(self, update):
         """Initialize the Exception."""
         self.update = six.text_type(update)
 
+    def __str__(self):
+        return str(self.__unicode__())
```

Comment:
I am not sure that this will work in Py2.7 when `self.__unicode__()` returns a true unicode with more than just ascii chars.

I would prefer providing the same method `self.__unicode__()' under both names: `__unicode__` and `__str__`. This makes no difference in Python 3 and in Python 2.7, we don't need exception text in bytes so I think that's ok to return unicode object from `__str__` method.

What do you think?

Reply:
> I am not sure that this will work in Py2.7 when self.__unicode__() returns a true unicode with more than just ascii chars.

@frenzymadness I've no idea what problem would happen. I tried to recall my memory and I haven't remember a failure case related to issue.

> providing the same method self.__unicode__() under both names

Sorry, I don't understand what it means. Can you give an example?

Reply:
> Sorry, I don't understand what it means. Can you give an example?

I mean something like `__str__ = __unicode__` which do basically the same as your code but without converting the result of `__unicode__` back into `str` which does nothing in Python 3 and can cause problems in Python 2.